resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.sh

```diff
@@ -8,19 +8,8 @@
 source ${TOOL_ANALYZE_BAM_HEADER}
 getRefGenomeAndChrPrefixFromHeader ${FILENAME_TUMOR_BAM} # Sets CHR_PREFIX and REFERENCE_GENOME
 
-## Getting tumor and control header
-VCF_TUMOR_HEADER_COL=`samtools view -H ${FILENAME_TUMOR_BAM} | grep -P "^@RG" | perl -ne 'chomp; @s=split(/\t/, $_) ; map{if($_=~/SM:/){$_=~/SM:(.*)/; print "$1\n"}} @s;' | sort | uniq`
```

Comment:
I hope the column names will always be the same as the BAM basename and we do not need to reanalysis any ICGC BAM files :)

Reply:
Are you getting unsure about this? I am wondering whether we will have to reanalyse the ICGC BAMs with this old workflow. We might check with @buchhalt to hear what he thinks.